controllers/tools/surveys/lib/percentages.js

```diff
@@ -3,10 +3,9 @@ const partition = require('lodash/partition');
 
 module.exports = function percentagesFor(responses) {
     const [yes, no] = partition(responses, ['choice', 'yes']);
-
     return {
         yesCount: yes.length,
         noCount: no.length,
-        percentageYes: Math.round((yes.length / responses.length) * 100),
+        percentageYes: ((yes.length / responses.length) * 100).toFixed(1),
```

Comment:
This is because the numbers are typically the same month to month, so adding in a decimal place gives a bit more to differentiate with.